src/github.com/matrix-org/go-neb/services/slackapi/slackapi.go

```diff
@@ -53,6 +56,20 @@ func (s *Service) OnReceiveWebhook(w http.ResponseWriter, req *http.Request, cli
 	w.WriteHeader(200)
 }
 
+// Register joins all configured rooms
+func (s *Service) Register(oldService types.Service, client *matrix.Client) error {
+	for _, mapping := range s.Hooks {
+		if _, err := client.JoinRoom(mapping.RoomID, "", ""); err != nil {
+			log.WithFields(log.Fields{
+				log.ErrorKey: err,
+				"room_id":    mapping.RoomID,
+				"user_id":    client.UserID,
+			}).Error("Failed to join room")
+		}
+	}
+	return nil
```

Comment:
You need to tell the caller what the webhook endpoint is. See https://github.com/matrix-org/go-neb/blob/master/src/github.com/matrix-org/go-neb/services/travisci/travisci.go#L245